src/utils/randomNumberGenerator.js

```diff
@@ -10,4 +10,7 @@ module.exports = {
 			.toString('hex'), 16);
 		return (randomInt % (max + 1 - min)) + min;
 	},
+	randomStringAsBase64Url: (size) => {
+		return crypto.randomBytes(size).toString('base64').replace(/\//g,'_').replace(/\+/g,'-');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A space is required after ','.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5697327)